### PR TITLE
Strengthen some normative statements in schema (wrong branch)

### LINKED ISSUE #868: Strengthen some normative statements in schema
> The ID used for cross-referencing to this party from other sections of the release. This field **may** be built with the following structure {identifier.scheme}-{identifier.id}(-{department-identifier}).

MAY -> SHOULD

> The maximum duration of this period in days. A user interface can collect or display this data in months or years as appropriate, and then convert it into days when storing this field. This field can be used when exact dates are not known. If a startDate and endDate are set, this field, if used, **should** be equal to the difference between startDate and endDate. Otherwise, if a startDate and maxExtentDate are set, this field, if used, **should** be equal to the difference between startDate and maxExtentDate.

SHOULD -> MUST

> The goods, services, and any intangible outcomes in this contract. Note: If the items are the same as the award do not repeat.

Change second clause to either SHOULD or MUST statement.

Follow-up to #855

### PR DESCRIPTION
**Pull request by ColinMaudry**
Closes #868 

> 
> > The goods, services, and any intangible outcomes in this contract. Note: If the items are the same as the award do not repeat.
> 
> Change second clause to either SHOULD or MUST statement.

I went for "should", as I don't believe repeating the items in the contract is harmful, just potentially confusing.

### CODE REVIEW

**Comment by jpmckinney** [comment]
@ColinMaudry If you create the branch from this repo (rather than your fork) the tests should pass. A few other notes:

* Please change the base branch to 1.2-dev for all new PRs (I've edited your existing PRs).
* Add an entry to `changelog.md` (I forgot to do this for #1067)
* Set the PRs milestone to `1.2.0`
* If you edit the release schema, you'll need to run `python utils/make_dereferenced_release_schema.py`
* If you add a field to the release schema, you'll need to run `python utils/make_versioned_release_schema.py`

**Comment by jpmckinney** [comment]
I've updated the instructions here: https://ocds-standard-development-handbook.readthedocs.io/en/latest/standard/contributing.html#proposing-changes

File: schema/release-schema.json [hunk 1]
```diff
@@ -685,7 +685,7 @@
         },
         "items": {
           "title": "Items contracted",
-          "description": "The goods, services, and any intangible outcomes in this contract. Note: If the items are the same as the award do not repeat.",
+          "description": "The goods, services, and any intangible outcomes in this contract. Note: you should not repeat the items if they are the same as the award.",
```
Comment (jpmckinney) [inline schema/release-schema.json]:
```suggestion
          "description": "The goods, services, and any intangible outcomes in this contract. If the contract's items are identical to the related award's items, they should not be repeated here.",
```

**Comment by jpmckinney** [comment]
@ColinMaudry Please update the changelog, and also run `python utils/make_dereferenced_release_schema.py`.

**Comment by ColinMaudry** [comment]
Replaced this PR with https://github.com/open-contracting/standard/pull/1112

**Pull request closed without merging**